src/components/views/messages/JumpToDatePicker.tsx

```diff
@@ -85,6 +85,7 @@ const JumpToDatePicker: React.FC<IProps> = ({ ts, onDatePicked }: IProps) => {
                 onInput={onDateValueInput}
```

Comment:
@Aman-Godara Is this PR meant to be in draft? Draft means you're still working on it and it's not ready for review yet.

Reply:
I was trying to test this on my local machine. The coding part is done unless testing fails.